bridges/evm/test/Bridge.test.js

```diff
@@ -195,53 +195,60 @@ contract("Bridge", ([_, owner, alice, bob]) => {
     beforeEach(async () => {
       this.bridge = await Bridge.new([]);
       await this.bridge.setOracleState(
-        "34", // _blockHeight
-        "0xF006784E23E7A0B48B550B486828AB9F14B74852D3168CE7FF19AE2F8DFE65AF", // _oracleIAVLStateHash
+        "14237", // _blockHeight
+        "0xB1AF69A024BC30E241B0C2F60599DEF30E2E788604390FCEE464A30F1D3BFA6A", // _oracleIAVLStateHash
       );
     });
 
     it("should not accept unrelayed block", async () => {
       await expectRevert(
         this.bridge.verifyOracleData(
-          "9999", // _blockHeight
-          ["band test", 1, "030000004254436400000000000000", 4, 4], // _requestPacket
-          ["band test", 1, 4, 1587734008, 1587734012, 1, "d8720b0000000000"], // _responsePacket
-          "326", // _version
+          "999999", // _blockHeight
+          ["beeb", 1, "0x030000004254436400000000000000", 4, 4], // _requestPacket
```

Comment:
We need to change client id look more professional? @sorawit

Reply:
I think this is ok. nothing rude or unappropriate.